recipes/minizip/build.sh

```diff
@@ -0,0 +1,16 @@
+
+
+cmake -S . -B build \
+  -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+  -DCMAKE_INSTALL_LIBDIR="$PREFIX/lib" \
+  -DBUILD_SHARED_LIBS=ON \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DMZ_BUILD_TESTS=ON \
+  -DMZ_LIBCOMP=OFF \
+  -DMZ_ZLIB=ON \
+  -DCMAKE_INSTALL_INCLUDEDIR=include/minizip \
+  -DMZ_FORCE_FETCH_LIBS=OFF
+
+cmake --build build
+
```

Comment:
I don't recall of they run test in that spec file I posted. Do we have a ctest or make check to run here?